src/support_enumeration.jl

```diff
@@ -0,0 +1,287 @@
+#=
+Compute all mixed Nash equilibria of a 2-player (non-degenerate) normal
+form game by support enumeration.
+
+Julia version of QuantEcon.py/support_enumeration.py
+
+Authors: Daisuke Oyama, Zejin Shi
+
+References
+----------
+B. von Stengel, "Equilibrium Computation for Two-Player Games in
+Strategic and Extensive Form," Chapter 3, N. Nisan, T. Roughgarden, E.
+Tardos, and V. Vazirani eds., Algorithmic Game Theory, 2007.
+=#
+
+"""
+    support_enumeration(g::NormalFormGame{2})
+
+Compute mixed-action Nash equilibria with equal support size for a
+2-player normal form game by support enumeration. For a
+non-degenerate game input, these are all Nash equilibria.
```

Comment:
Question about English:
Should it be better to say "these are all the Nash equilibria" ("the" added) or "all the Nash equilibria are enumerated"? @cc7768 @sglyon 
(The same applies to the Python version.)

(In a degenerated game, there may be equilibria that the routine misses to return.)